designs/administrator/future-openshift/network-policy-form/index.md

```diff
@@ -0,0 +1,53 @@
+---
+parent: Administrator
+version: 4.x
+---
+
+# Network policy form
+
+# Background
+Setting Network policies is a critical part of the security of applications on a cluster. In OCP 4.7 the only way to manage Network policy in the console is to edit the yaml of these resources. It's a common and important enough operation that we should make this easier for administrators to work with.
+
+## Goal
+
+Add the ability to create/update/delete NetworkPolicy resources through the OpenShift console in a way that is easier than directly editing the yaml representation of these resources.
+
+## Designs for 4.9 release
+
+This initial concept is a form based solution that compliments the current YAML editor. This allows users to switch between the form and YAML views so they may better understand the YAML output.
+
+### Create Network Policy introduction
+
+* The policy will pre-fill the namespace based on the Project selector at the top.
```

Comment:
In the mocks provided the namespace in the form (default) is different than the namespace in the project selector (my-project). Also, when we prefill fields I believe the text is black rather than the placeholder grey.